docs/source/buildingWindows.rst

```diff
@@ -0,0 +1,74 @@
+
+Interactively building SimpleITK on Windows
+###########################################
+
+This page describes how to interactively build a SimpleITK Superbuild on
+Windows.  To build SimpleITK we use CMake to generate a Visual Studio project,
+and Visual Studio to compile the C/C++ code.
+
+The Windows version of CMake includes a graphical-user-interface (GUI) version
+of the program, in addition to the standard command-line CMake.  Here is what
+the CMake GUI looks like:
+
+.. image:: ../images/cmake-win-1.png
+  :width: 400
+  :alt: Windows CMake GUI
+
+In the above example, the source directory is 'C:/SimpleITK', and the
+build directory is 'C:/SimpleITK-build'.  It is best to place the source
+and build directories at the root of a volume to minimize path name lengths.
+
+After the user specifies the **source** and **build** directories, CMake will
+ask to select which compiler generator to use.  The options are determined by
+what versions of Visual Studio are installed on the system.  The user must
+specify **x64** for the platform version, since the default option (x86) is
+not supported by SimpleITK.
+
+.. image:: ../images/cmake-win-2.png
+  :width: 400
+  :alt: Visual Studio Selection
+
+The next steps are the configuration and generation processes.  By clicking
+the **Configure** button, cmake reads the CMakeLists.txt file and sets up all
+the variables in the CMake cache (CMakeCache.txt).  Then when the **Generate**
+button is clicked, CMake creates the Visual Studio build system.
+
+The left image of the above figure is the result of the configure process, and
+on the right is the result of the generation process.
+
+Note for the **CMAKE_CXX_FLAGS** and **CMAKE_C_FLAGS** variables, we have
```

Comment:
Any way to highlight this? It isn't a "nice to have" it's a must have. Without the /MP flag the developer will die of old age before the SimpleITK build finishes.

Reply:
shouldn't we be setting that automatically?

Reply:
In general /MP does not behave well with CMake super build projects as there is not global process manager, each sub-project runs /MP with out regards to the parent. Too many builds can run on CI services and memory issues can occur.

Reply:
Got it. So we leave the instructions as is, and it is up to the person compiling it on windows to do the right thing.